sdks/RepoToolset/tools/Tests.props

```diff
@@ -6,6 +6,11 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsPerformanceTestProject)' == ''">
+    <IsPerformanceTestProject>false</IsPerformanceTestProject>
+    <IsPerformanceTestProject Condition="$(MSBuildProjectName.EndsWith('.PerfTests')) or $(MSBuildProjectName.EndsWith('.PerformanceTests'))">true</IsPerformanceTestProject>
```

Comment:
Do we need to support both `PerfTests` and `PerformanceTests`?

Reply:
Why would we not do this?

Reply:
It is easier to say there is one recognized pattern and to stick to it. `PerfTests` seems sufficient enough.

Reply:
Okay...